sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TTLState.scala

```diff
@@ -240,9 +248,10 @@ abstract class CompositeKeyTTLStateImpl(
       val expirationMs = kv.key.getLong(0)
       StateTTL.isExpired(expirationMs, ttlExpirationMs)
     }.foreach { kv =>
-      val groupingKey = kv.key.getBinary(1)
-      val userKey = kv.key.getBinary(2)
-      numRemovedElements += clearIfExpired(groupingKey, userKey)
+      // TODO get the right numFields
```

Comment:
what are we referring to here ?

Reply:
Sorry, I resolved this already but forgot to remove.